src/main/resources/liquidbounce.mixins.json

```diff
@@ -11,8 +11,6 @@
     "minecraft.block.MixinCobwebBlock",
     "minecraft.block.MixinFluidBlock",
     "minecraft.block.MixinSnowBlock",
-    "minecraft.block.MixinSlimeBlock",
-    "minecraft.block.MixinPowderSnowBlock",
```

Comment:
Is there a reason you removed these two mixin files?

Reply:
No, I removed it by accident (idk how) when I adding a new mixin file